src/Diagnostics.RuntimeHost/Controllers/ControllerBase.cs

```diff
@@ -53,7 +54,7 @@ protected OperationContext<TResource> PrepareContext<TResource>(TResource resour
                 resource,
                 DateTimeHelper.GetDateTimeInUtcFormat(startTime).ToString(DataProviderConstants.KustoTimeFormat),
                 DateTimeHelper.GetDateTimeInUtcFormat(endTime).ToString(DataProviderConstants.KustoTimeFormat),
-                isInternalRequest,
+                isInternalRequest || forceInternal,
```

Comment:
Basically these calls are coming from geomaster but they are still internal requests essentially. I am open to another option, this was the quickest way for me. 

Right now this API is always internal, at least in my opinion, even if it doesn't have the applens header, which it won't. I could add the applens healder for all calls to this API, but that seems incorrect since it isn't really coming from applens. Thoughts?

Reply:
I see. So this flag is set to true in the API we are making for ASC . 
Thats fine then.
Another approach could be we add a user-agent (or something in the headers) at the caller side(geomaster, applens controller) for AppLens and Azure Support Center and then decide whether its internal or external.
Same complexity .. but it can help logging in future.

Reply:
Yeah i was thinking user agent as well, I think that is a better approach

Reply:
I am going to leave this for now because it doesn't seem I can set header as is in geomaster, want to keep changes there to a minimum.